bluetoothServicesComponent/modem_info_service.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __MODEM_INFO_SERVICE_H_
+#define __MODEM_INFO_SERVICE_H_
+
+#define MODEM_INFO_SERVICE_UUID "180A"
```

Comment:
The device information service doesn't provide a way to list the IMEI, so I think we should either provide both the device information service and another service for providing the IMEI or we should just provide a single non-standard service which provides both the IMEI and serial number.